.github/workflows/unit-tests-operator.yaml

```diff
@@ -20,6 +21,7 @@ on:
       - '/go.sum'
       - 'operator/**'
       - 'api/types/**'
+      - 'gen/**'
       - 'lib/tbot/**'
```

Comment:
Sadly this should be `lib/**` I think as the `gen/` dependency was added by a `lib/something` edition, which was likely a dependency of `lib/tbot`.

Reply:
Yeah, I was also thinking we should just run operator tests on any .go source code change to be safe. It runs fast anyway. Will fix later (after release).